packages/cletus/src/operations/dba.tsx

```diff
@@ -604,6 +604,164 @@ export const data_index = operationOf<
   },
 });
 
+export const data_import = operationOf<
+  { name: string; glob: string; chunkSize?: number; overlap?: number },
+  { imported: number; updated: number; skipped: number; libraryKnowledgeUpdated: boolean }
+>({
+  mode: 'create',
+  status: ({ name, glob }) => `Importing ${name} from ${glob}`,
+  analyze: async ({ name, glob }, { config, cwd }) => {
+    const type = getType(config, name);
+    const { glob: globLib } = await import('glob');
+    const files = await globLib(glob, { cwd });
+    
+    return {
+      analysis: `This will import ${type.friendlyName} records from ${files.length} file(s) matching "${glob}". Files will be processed in chunks with AI extraction.`,
+      doable: true,
+    };
+  },
+  do: async ({ name, glob, chunkSize = 4000, overlap = 200 }, ctx) => {
+    const type = getType(ctx.config, name);
+    const dataManager = new DataManager(name);
+    await dataManager.load();
+    
+    // Find files matching glob
+    const { glob: globLib } = await import('glob');
+    const fs = await import('fs/promises');
+    const path = await import('path');
+    const files = await globLib(glob, { cwd: ctx.cwd });
+    
+    ctx.chatStatus(`Found ${files.length} files to process`);
+    
+    // Build type schema description for AI
+    const typeSchema = buildTypeSchemaDescription(type);
+    
+    let allExtractedRecords: Record<string, any>[] = [];
+    
+    // Process each file
+    for (let fileIndex = 0; fileIndex < files.length; fileIndex++) {
```

Comment:
See clerk.tsx for how files are searched, filtered by type, then use parallel processFile calls. Add support for transcribeImages which means images should be considered for import. No summary or describing images though for import.